src/Uno.UI.RuntimeTests/IntegrationTests/common/TestServices.KeyboardHelper.cs

```diff
@@ -121,7 +121,13 @@ public static class KeyboardHelper
 			public static void PressKeySequence(string keys, UIElement element = null)
 			{
 #if !NETFX_CORE
-				if (string.IsNullOrEmpty(keys) || element == null)
+				if (string.IsNullOrEmpty(keys))
+				{
+					return;
+				}
+
+				element ??= FocusManager.GetFocusedElement() as UIElement;
```

Comment:
@dr1rrb I remember we had a conversation about trying not to touch this file. Does this change seem reasonable enough? The tests seem to work on windows with element = null.

Reply:
@dr1rrb The current code returns from the method without doing anything for null element, however, there are lots of tests that are passing null element (by omitting the argument and using the default parameter value). So I think this method specifically doesn't match WinUI already. Otherwise, WinUI tests would be broken.